app/queries/get_linkables.rb

```diff
@@ -33,7 +33,7 @@ def call
           state_fallback_order: [:published, :draft]
         )
 
-        Edition.where(id: edition_ids).map do |edition|
+        Edition.with_document.includes(:document).where(id: edition_ids).map do |edition|
```

Comment:
Ah yes - I didn't realise Active Record did that when you have a join (TIL) sorry about that